Classes/Service/VarnishBanService.php

```diff
@@ -66,10 +61,14 @@ public function initializeObject()
         array_walk($varnishUrls, function (&$varnishUrl) {
             $varnishUrl = rtrim($varnishUrl, '/');
         });
-        $this->varnishProxyClient = new ProxyClient\Varnish($varnishUrls);
-        $this->varnishProxyClient->setDefaultBanHeader('X-Site', $this->tokenStorage->getToken());
+        $httpDispatcher = new FOS\HttpCache\ProxyClient\HttpDispatcher($varnishUrls, '');
+        $options = [
+            'default_ban_headers' => [
+                "X-Site" => $this->tokenStorage->getToken()
+            ]
+        ];
+        $this->varnishProxyClient = new Varnish($httpDispatcher, $options);
```

Comment:
The `Varnish` class is not imported as far as I can see.

```suggestion
        $this->varnishProxyClient = new ProxyClient\Varnish($httpDispatcher, $options);
```